app/components/elements/forms/base_collection_component.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Elements
+  module Forms
+    # Base component for a set of inputs for a collection, e.g. checkboxes or radio buttons
+    class BaseCollectionComponent < ApplicationComponent
```

Comment:
I hate to ask this, but what is a collection in this context? I ask because `collection` already means something in the H3 domain model. If this means a form element that can have multiple values, I wonder if we might use a different term.

Reply:
Open to discussion on this. This is the term from the Rails side of things.

Reply:
@justinlittman Do you have a long-story-short sense of e.g. how `select` and `collection_select` are different?

Reply:
I'm not sure they are significantly different for selects. For checkboxes and radios, they make building sets of them much easier.

Reply:
@justinlittman OK. How about we use the term `InputCollection` (`input_collection`)? Would that be a net plus, net minus, or net zero in terms of clarity?

Reply:
I like that.